adhocracy-plus/templates/footer_upper.html

```diff
@@ -6,12 +6,27 @@
         <span>{% trans 'Responsible for the content of this page:' %}</span>
     </div>
 
-    <nav class="navbar justify-content-center">
+    <nav class="navbar navbar-expand-lg justify-content-center">
         <span class="navbar-brand">{{ ORGANISATION.name }}</span>
         <ul class="navbar-nav">
             <li class="nav-item">
-              {% url 'organisation-imprint' organisation_slug=ORGANISATION.slug as organisation_imprint_url %}
-              <a href="{{ organisation_imprint_url }}" class="header-lower__nav-link"><span class="{% if request.path == organisation_imprint_url %}is-active{% endif %}">{% trans 'Imprint' %}</span></a>                    </li>
+                {% url 'organisation-imprint' organisation_slug=ORGANISATION.slug as organisation_imprint_url %}
+                <a href="{{ organisation_imprint_url }}" class="nav-link header-lower__nav-link"><span class="{% if request.path == organisation_imprint_url %}is-active{% endif %}">{% trans 'Imprint' %}</span></a>
+            </li>
+            <li class="nav-item">
+                {% url 'organisation-data-protection' organisation_slug=ORGANISATION.slug as organisation_data_protection_url %}
+                <a href="{{ organisation_data_protection_url }}" class="nav-link header-lower__nav-link"><span class="{% if request.path == organisation_data_protection_url %}is-active{% endif %}">{% trans 'Data protection policy' %}</span></a>
```

Comment:
This adds "is-active" when you are on the site, but the bootstrap way would be to add "active" to the li, but that also doesn't change the styling. So I would remove the active-class for now?

Reply:
yeah I think thats fine!